### ISSUE TITLE
[Bug]: Preview looks all messed up with Bun v1.0.15

### ISSUE THREAD

**Issue by wallpants** [wallpants/github-preview.nvim#254]
### What happened?

<img width="888" alt="Screenshot 2023-12-05 at 11 15 03 p m" src="https://github.com/wallpants/github-preview.nvim/assets/47203170/81459fcf-e286-4894-aa12-1bfee844e22e">

### How were you expecting the plugin to behave?

I was expecting it not to look like that

### :checkhealth

```shell
github-preview.nvim ~
- platform: macos
- git-commit: 75c6697
- OK bun: 1.0.15
```

**Comment by wallpants** [comment]
Thank you for reporting this issue, we'll look into it.

 For now please downgrade to Bun v1.0.14 if you want to use this plugin.

**Comment by Jarred-Sumner** [comment]
oh no, what did we break?

**Comment by wallpants** [comment]
@Jarred-Sumner not sure, but y'all fixed it with v1.0.19.

Thanks for your work on Bun. Exciting stuff.
